inst/htmlwidgets/lib/profvis/profvis.js

```diff
@@ -1730,8 +1772,26 @@ profvis = (function() {
       };
     });
 
+    var aggregateTreeMem = function(root) {
+      var nodes = [root];
+      var sum = 0;
+
+      while(nodes.length > 0) {
+        var node = nodes.pop();
+        sum = node.meminc ? sum + node.meminc : sum;
+
+        if (node.children) {
+          node.children.forEach(function (child) {
+            nodes.push(child);
```

Comment:
I know, fun times. However, I had to remove this traversal and integrate with `consolidateTree`. At the time I didn't realize that there is a consolidation process that gets rid of duplicate call-stacks, so basically the original pr was missing a bunch of incremental data and the memory visualizations were not accurate. I spent a good part of the day validating cases and running the raw `Rprof` data through R (link-by-line) to validate the memory allocations are correct. So far so good.